### ISSUE TITLE
Clear app-build when remix build is run

### ISSUE THREAD

**Issue by NejcZdovc** [remix-run/remix#83]
When I rename one route old route was still in `app-build`. Because of that when I did `vercel` old route was pushed to the server and vercel returned error that that old route was not in manifest. Probably what we need is that we clear that folder on every build so that old files are removed.

**Comment by Aprillion** [comment]
Workaround I use in my project (Cloudflare Workers) until this is fixed: https://github.com/Aprillion/stampy-ui/commit/ca95346d65090b4558587ba955b54c8ce602963a


```diff
package.json
-"build": "cross-env NODE_ENV=production remix build",
+"build": "rimraf public/build && cross-env NODE_ENV=production remix build",
```

**Comment by andrew-secret** [comment]
@Aprillion  Maybe you can help me. I'm trying to set up a remix project with Cloudflare workers. Each time I spin up the dev server with Miniflare I always see that my build directory is full of hashed files which are generated on the fly on each file change. Is this related?

**Comment by brophdawg11** [comment]
This should be fixed with #3061, which will be in next week's release 👍
